package.json

```diff
@@ -7,9 +7,9 @@
     "build": "vitepress build docs"
   },
   "devDependencies": {
-    "@warp-ds/component-classes": "1.0.0-alpha.59",
+    "@warp-ds/component-classes": "1.0.0-alpha.60",
```

Comment:
I saw the [last commit](https://github.com/warp-ds/component-classes/actions/runs/4934098671/jobs/8818734826) didn't get released but we'll get there sooner or later with the next commits :D